torch/_dynamo/polyfills/tensor.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any
+
+import torch
+
+from ..decorators import substitute_in_graph
+
+
+@substitute_in_graph(  # type: ignore[arg-type]
+    torch.Tensor._make_subclass
+)
+def make_subclass(
+    cls: type[Any], data: torch.Tensor, requires_grad: bool = False, **kwargs: Any
+) -> Any:
+    # This is a rough approximation of `THPVariable_make_subclass`. It should
+    # suffice for most of Dynamo tracing purposes.
+    assert len(kwargs) == 0, "_make_subclass only supports requires_grad as keyword arg"
+    data = data.detach()
+
+    # Avoid unnecessary `requires_grad` mutation, which isn't supported in Dynamo.
+    if data.requires_grad != requires_grad:
+        data.requires_grad = requires_grad
```

Comment:
uhh, can we just assert requires_grad=False? Dynamo's not good with intermediate tensors that require grad (hence `Tensor.requires_grad_` graph breaking, which is probably what you ran into)

Reply:
Hmm but asserting would lead to a graph break as well, and it's probably going to be more confusing than the existing graph break?